lib/rules/test-case-property-ordering.js

```diff
@@ -37,18 +37,21 @@ module.exports = {
         utils.getTestInfo(context, ast).forEach(testRun => {
           [testRun.valid, testRun.invalid].forEach(tests => {
             (tests || []).forEach(test => {
-              const keys = (test.properties || []).map(item => utils.getKeyName(item));
+              const keys = (test.properties || []).map(utils.getKeyName);
 
               for (let i = 0, lastChecked; i < keys.length; i++) {
                 const current = order.indexOf(keys[i]);
 
                 // current < lastChecked to catch unordered;
                 // and lastChecked === -1 to catch extra properties before.
                 if (current > -1 && (current < lastChecked || lastChecked === -1)) {
+                  let orderMsg = order.filter(item => keys.indexOf(item) > -1);
+                  orderMsg = orderMsg.concat(lastChecked === -1 ? [keys[i - 1]] : []);
```

Comment:
Hmm, that's true. I think it would be fine to say a specific order in the error message, but also allow a different order for the unknown keys.